pandas/core/panel.py

```diff
@@ -679,6 +679,8 @@ def _combine(self, other, func, axis=0):
             return self._combine_frame(other, func, axis=axis)
         elif np.isscalar(other):
             return self._combine_const(other, func)
+        else:
+            raise NotImplementedError('This method has not been implemented yet. ')
```

Comment:
@jreback any idea on how to access the op-method-name that's not implemented? 

`func` returns `<function na_op at 0x10358dde8>` and `func.__name__` returns `na_op`. Neither of these is something I expected, e.g. `add`, `mul`, etc. What is `na_op` BTW?